resources/migrations/000_migrations.yaml

```diff
@@ -3939,4 +3939,52 @@ databaseChangeLog:
                   type: text
               - column:
                   name: ended_at
-                  type: DATETIME
\ No newline at end of file
+                  type: DATETIME
+  - changeSet:
+      id: 69
+      author: senior
+      comment: 'Added 0.27.0'
+      remarks: 'Add columns to the pulse table for alerts'
+      changes:
+        - addColumn:
+            tableName: pulse
+            columns:
+              - column:
+                  name: alert_condition
+                  type: varchar(254)
+                  remarks: 'Condition (i.e. "rows" or "goal") used as a guard for alerts'
+                  constraints:
+                    nullable: true
```

Comment:
things in Liquibase are nullable by default just like when writing DDL statements in SQL. So you don't need to put  `nullable: true` in there